test/extended/networking/ipsec.go

```diff
@@ -632,6 +653,117 @@ var _ = g.Describe("[sig-network][Feature:IPsec]", g.Ordered, func() {
 	})
 })
 
+var _ = g.Describe("[sig-network] IPsec resilience", g.Ordered, func() {
+	oc := exutil.NewCLIWithPodSecurityLevel("ipsec", admissionapi.LevelPrivileged)
+	f := oc.KubeFramework()
+	var ipsecMode v1.IPsecMode
+
+	InOVNKubernetesContext(func() {
+		g.BeforeAll(func() {
+			var err error
+			ipsecMode, err = getIPsecMode(oc)
+			o.Expect(err).NotTo(o.HaveOccurred())
+		})
+
+		g.It("check pod traffic are working across nodes", func() {
+			if ipsecMode != v1.IPsecModeFull {
+				e2eskipper.Skipf("cluster is configured with IPsec %s mode, so skipping the test", ipsecMode)
+			}
+			g.By("creating test pods")
+			pods := createWebServerPods(oc, f.Namespace.Name)
+			g.By("checking crossing connectivity over the pods")
+			checkPodCrossConnectivity(pods)
+		})
+
+		g.It("check pod traffic are working across nodes after ipsec daemonset restart", func() {
+			if ipsecMode != v1.IPsecModeFull {
```

Comment:
why not do this for other modes?

Reply:
the previous tested is valid for all ipsec modes, but this one is valid for full mode, added a comment in the code, hope that helps.